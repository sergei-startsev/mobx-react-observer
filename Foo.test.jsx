import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { createSerializer } from 'enzyme-to-json';
import FooWrapped, { Foo } from './Foo.jsx';

Enzyme.configure({ adapter: new Adapter() });
expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));

describe('Foo', () => {
  test('shallow', () => {
    const component = shallow(<Foo />);
    expect(component).toMatchSnapshot();
  });
})

describe('Foo Wrapped', () => {
  test('shallow', () => {
    const component = shallow(<FooWrapped store={{}}/>);
    expect(component).toMatchSnapshot();
  });
})