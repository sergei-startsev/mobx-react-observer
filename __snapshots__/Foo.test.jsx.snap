// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foo Wrapped shallow 1`] = `
<Foo
  store={Object {}}
/>
`;

exports[`Foo shallow 1`] = `
<div>
  Test
</div>
`;
